app/helpers/ukims_helper.rb

```diff
@@ -22,8 +22,15 @@ def trader_scheme_body
     if after_cut_off_date?
       "If you are moving goods into Northern Ireland which are for sale to, or final use by, end consumers located in the UK and you are authorised under the UK Internal Market Scheme, then you may declare your goods as being 'not at risk' where the requirements are met. A 'not at risk' good entering Northern Ireland from Great Britain will not be subject to duty."
     else
-      "If you were moving goods into Northern Ireland which are for sale to, or final use by, end consumers located in the UK and you were authorised under the UK Trader Scheme or the UK Internal Market Scheme, then you may declare your goods as being 'not at risk' where the requirements are met. A 'not at risk' good entering Northern Ireland from Great Britain will not be subject to duty.
-       Please note that UK Internal Market scheme trades cannot benefit from expanded processing rules before 30 September 2023. Trades before this date will use the UK Trader Scheme rules on processing"
+      "If you were moving goods into Northern Ireland which are for sale to, or final use by, end consumers located in the UK and you were authorised under the UK Trader Scheme or the UK Internal Market Scheme, then you may declare your goods as being 'not at risk' where the requirements are met. A 'not at risk' good entering Northern Ireland from Great Britain will not be subject to duty."
+    end
+  end
+
+  def trader_scheme_body_extra_text
```

Comment:
what about renaming it to something more evident?

maybe ` internal_market_scheme_note`